src/cascadia/WindowsTerminal/resource.h

```diff
@@ -3,20 +3,31 @@
 // Used by WindowsTerminal.rc
 //
 #define IDI_APPICON 101
-#define IDS_ERROR_DIALOG_TITLE 105
-#define IDS_HELP_DIALOG_TITLE 106
-#define IDS_ERROR_ARCHITECTURE_FORMAT 110
-#define IDS_X86_ARCHITECTURE 111
-#define IDS_AMD64_ARCHITECTURE 112
-#define IDS_ARM64_ARCHITECTURE 113
-#define IDS_ARM_ARCHITECTURE 114
-#define IDS_UNKNOWN_ARCHITECTURE 115
+#define IDI_APPICON_HC_B 102
+#define IDI_APPICON_HC_W 103
+
+#define IDI_APPICON_DEV 104
+#define IDI_APPICON_DEV_HC_B 105

```

Comment:
so i am just now noticing that the DEV_HC icons do not actually have the "DEV" badge, even though the SVG has it.  that appears to be a bug left over from when i monkeyed with the icons.  i just tried running the script, and the icons got better... i wonder how i overlooked doing this?  but can i request that you slip this in, while you're at it?

```
.\Generate-TerminalAssets.ps1 -path .\Terminal_Dev.svg -HighContrastPath .\Terminal_Dev_HC.svg -Destination .\images-dev
```

Reply:
Sure! Good catch.